onebusaway-android/src/main/java/org/onebusaway/android/ui/FeedbackActivity.java

```diff
@@ -0,0 +1,165 @@
+package org.onebusaway.android.ui;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.Toast;
+
+import org.apache.commons.io.FileUtils;
+import org.onebusaway.android.R;
+import org.onebusaway.android.app.Application;
+import org.onebusaway.android.nav.NavigationService;
+import org.onebusaway.android.nav.NavigationServiceProvider;
+import org.onebusaway.android.util.PreferenceUtils;
+
+import java.io.File;
+import java.io.IOException;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import static org.onebusaway.android.nav.NavigationService.LOG_DIRECTORY;
+
+public class FeedbackActivity extends AppCompatActivity {
+
+    public static final String TAG = "FeedbackActivity";
+
+    private String mAction = null;
+    private ImageButton dislikeButton;
+    private ImageButton likeButton;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_feedback);
+        setTitle(getResources().getString(R.string.feedback_label));
+
+        Intent intent = this.getIntent();
+        CheckBox sendLogs = (CheckBox) findViewById(R.id.feedback_send_logs);
+        if (intent != null) {
+            mAction = intent.getExtras().getString("CallingAction");
```

Comment:
"CallingAction", "no" and "yes" (and any other intent extras and actions) need to be defined as public static final variables - let's put them in FeedbackActivity and reference them always from here.

Reply:
We have modified the code to reference the values as public static final variables